servers/republik/graphql/resolvers/_mutations/requestNewsletterSubscription.js

```diff
@@ -16,6 +16,10 @@ module.exports = async (_, args, context) => {
     t
   } = context
 
+  if (['COVID19'].includes(name)) {
```

Comment:
shouldn't it be the opposite?

Reply:
yes, thanks! I didn't check the suggestion of @patrickvenetz well enough: https://github.com/orbiting/backends/pull/400#discussion_r398692374